packages/web-components/src/components/masthead/__stories__/cloud-masthead.stories.ts

```diff
@@ -62,6 +62,7 @@ export const Default = !DDS_CLOUD_MASTHEAD
                 platform-url="${ifNonNull(platformData.url)}"
                 selected-menu-item="${ifNonNull(selectedMenuItem)}"
                 ?has-contact="${hasContact}"
+                auth-method="cookie"
```

Comment:
Thanks for adding this new attribute/prop `auth-method` and made it available in the `cloud-masthead-container`. This is an enhancement, but I think it'll be nice to expose these Authentication method options as a knob select within the storybook preview.

Also, we should note this attribute/prop in the `README.stories.mdx` file as well.

Reply:
Updated the README. I'll need to think more on adding a knob to control auth method, since it currently won't work since the method needs to fire on load.

Reply:
@huntermacd No worries! The auth method knob is just a fancy addition, I don't think it's a blocker for this PR. As long as we have this attribute accounted for in the dev README, I'm fine with it. 👍